run_everything.R

```diff
@@ -0,0 +1,55 @@
+
+# ============================================================================ #
+# Update data used for the forecasts
+# ============================================================================ #
+source("utils/get_us_deaths.R")
+
+
+
+# ============================================================================ #
+# load in function to prepare submissions
+# ============================================================================ #
+source("utils/format_submission.R")
+
+
+
+
+# update delays (not run)
+# source("rt-forecast/update-delays.R")
+
+
+# ============================================================================ #
+# Run all models, format submission, do plotting
+# ============================================================================ #
+
+
+target_dates <- seq(ymd('2020-05-15'), ymd(Sys.Date()), by = '1 day')
+
+for (t in target_dates) {
+  target_date <- t
```

Comment:
the command and control script shouldn't have any code that does anything in it. suggest moving to a script